src/GafferScene/TweakPlug.cpp

```diff
@@ -280,12 +280,12 @@ class NumericTweak
 	const std::string &m_tweakName;
 };
 
-void applyTweakInternal( TweakPlug::Mode mode, const ValuePlug *valuePlug, const std::string &tweakName, const InternedString &parameterName, IECore::CompoundData *parameters, TweakPlug::MissingMode missingMode )
+bool applyTweakInternal( TweakPlug::Mode mode, const ValuePlug *valuePlug, const std::string &tweakName, const InternedString &parameterName, IECore::CompoundData *parameters, TweakPlug::MissingMode missingMode )
 {
 	if( mode == TweakPlug::Remove )
 	{
 		parameters->writable().erase( parameterName );
-		return;
+		return true;
```

Comment:
Should we return false here if there isn't actually something to erase?
```suggestion
		return parameters->writable().erase( parameterName );
```

Reply:
Good call, changed in https://github.com/GafferHQ/gaffer/pull/3699/commits/4c8dfa822777076457e4d59a6524aef177566d2b